karma.conf.js

```diff
@@ -13,19 +13,12 @@ const webpackTestConfig = merge(webpackConfig, {
     output: null,
     devtool: false,
     externals: {
-        $: {
-            commonjs: 'jquery',
-            amd: 'jquery',
-            root: '$'
-        },
         sinon: 'sinon'
     },
     resolve: {
         alias: {
             'test/underscore': path.resolve(__dirname + '/node_modules/underscore/underscore.js'),
             'utils/video': path.resolve(__dirname + '/test/mock/video.js'),
-            // Tests using jQuery: api-test, jwplayer-selectplayer-test, setup-test
-            jquery: path.resolve(__dirname + '/node_modules/jquery/dist/jquery.js'),
```

Comment:
do we need to update any test pages?

Reply:
There are only two pages in test and neither use `$`.